### PR TITLE
Remove manifest.json proxy and replace with frontend cache.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: components/Mediator.vue [hunk 1]
```diff
@@ -552,7 +556,68 @@ function updateHandlerWindow(handlerWindow) {
...
+      if(e.name === 'QuotaExceededError') {
+        // clear the whole cache
+        caches.delete(CACHE_NAME);
+        cacheStorage = null;
+        // use fetched response
+        return response.json();
+      }
+
+      // failed to fetch
+      console.error(e);
```
Comment (davidlehn) [inline components/Mediator.vue:605]: I think the common 404 path comes through here?  Could this be a bit more friendly than an error?  At least some short text saying why this is logged.  And in the 404 case, and maybe others, perhaps this is more of a warning than an error?

File: components/Mediator.vue [hunk 2]
```diff
@@ -552,7 +556,68 @@ function updateHandlerWindow(handlerWindow) {
...
+        // return cached response
+        return response.json();
+      }
+      // remove expired response from cache
+      await cacheStorage.delete(url);
+    }
+
+    try {
+      // fetch live response
+      response = await fetch(url);
```
Comment (davidlehn) [inline components/Mediator.vue:583]: Should this be using `getWebAppManifestFromUrl`?  (and handling the possible null result appropriately)

Comment (davidlehn) [reply components/Mediator.vue:583]: I see that api just returns data.  In any case, should this be using httpClient somehow?

Comment (dlongley) [reply components/Mediator.vue:583]: No, `httpClient` is only used in the degenerate case. I'd like to remove it entirely and just use fetch, but figured we may need to still support some old browser case.

File: components/Mediator.vue (hunk 1 above)
Comment (dlongley) [reply components/Mediator.vue:605]: I filed an issue to experiment with what to do here. See: #90.

**Pull request merged**